helix-view/src/editor.rs

```diff
@@ -141,6 +141,7 @@ pub struct Config {
     pub auto_info: bool,
     pub file_picker: FilePickerConfig,
     /// Shape for cursor in each mode
+    pub status_line: StatusLineConfig,
```

Comment:
Let's name this `statusline`, similar to `bufferline` in https://github.com/helix-editor/helix/pull/2759.

Reply:
Changed with c0a1870.